src/ui/page/dataverse/dataverse.tsx

```diff
@@ -124,7 +124,7 @@ export const dataverseItems: DataverseItemDetails[] = [
     type: 'zone',
     label: 'DS4I',
     description:
-      "Data Space for Investors (DS4I) is a private Zone created and maintained by OKP4 Team. DS4I's purpose is to present a simple and user-friendly Proof of Concept to demonstrate for potential investors how OKP4 protocol works based on simple governance rules.",
+      "Zone for Investors (DS4I) is a private Zone created and maintained by OKP4 Team. DS4I's purpose is to present a simple and user-friendly Proof of Concept to demonstrate for potential investors how OKP4 protocol works based on simple governance rules.",
```

Comment:
`DS4I` has to remain stay `Data Space for Investors`😄

Reply:
Missed that🙈 Fixed now!